src/highlight.c

```diff
@@ -262,7 +262,8 @@ static char *(highlight_init_both[]) = {
     "default link PmenuMatchSel PmenuSel",
     "default link PmenuExtra Pmenu",
     "default link PmenuExtraSel PmenuSel",
-    CENT("ComplMatchIns cterm=NONE", "ComplMatchIns gui=NONE"),
+    CENT("ComplMatchIns ctermfg=DarkGrey cterm=NONE",
+	 "ComplMatchIns guifg=DarkGrey gui=NONE"),
```

Comment:
You need to also add `ctermfg` to the second entry, otherwise `ctermfg` is not applied when `FEAT_GUI` is defined?

Also it may be inappropriate to set this by default. For `background=dark` it looks ok, but for `background=light` the text is a bit hard to notice.